### PR TITLE
fix: validate asset values are positive number for output operations

### LINKED ISSUE #294: Check if `/construction/preprocess` MA operation value is not empty is currently validated, if not, add that check

### PR DESCRIPTION
**Pull request by guido-ta**
# Description

Adds token asset value validation. It must be a positive number for output operations.
Also adds some logging calls.

Resolves #294

### CODE REVIEW

**Review by AlanVerbner — COMMENTED** [review]
No new tests for this check?

File: cardano-rosetta-server/test/e2e/construction/construction-preprocess-api.test.ts [hunk 1]
```diff
@@ -420,5 +417,91 @@ describe(CONSTRUCTION_PREPROCESS_ENDPOINT, () => {
         retriable: false
       });
     });
+
+    test('Should fail if MultiAsset value is negative', async () => {
```
Comment (AlanVerbner) [inline cardano-rosetta-server/test/e2e/construction/construction-preprocess-api.test.ts:421]: keep in mind this is for outputs only

File: cardano-rosetta-server/src/server/utils/cardano/operations-processor.ts [hunk 2]
```diff
@@ -17,27 +17,53 @@ import { isPolicyIdValid, isTokenNameValid } from '../validations';
 import { generateRewardAddress } from './addresses';
 import { getStakingCredentialFromHex } from './staking-credentials';
 
+const isNumeric = (value: string): boolean => /^\d+$/.test(value);
```
Comment (AlanVerbner) [inline cardano-rosetta-server/src/server/utils/cardano/operations-processor.ts:20]: don't we use a similar test somewhere else?

Comment (guido-ta) [reply cardano-rosetta-server/src/server/utils/cardano/operations-processor.ts]: good catch 👍

**Review by AlanVerbner — APPROVED** [review]
lgtm

**Pull request merged**